### PR TITLE
Finished designing editMenuItems

### PR DESCRIPTION
**Pull request by Pear99**

### CODE REVIEW

File: client/src/components/editMenuItems.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                        </div>
+                    </div>
+                    <Toppings></Toppings>
+                    <div>
+                        <input
+                            className="ml-4 mt-3 mr-1 w-25 form-control-sm"
+                            type="text"
+                            placeholder="Pris utan tillägg..."
+                        ></input>
+                        <span>{"(kr)"}</span>
```
Comment (steget) [inline client/src/components/editMenuItems.js:55]: Doesn't need to be seen as js
```suggestion
                        <span>(kr)</span>

```

File: client/src/components/editMenuItems.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import "../app.css";
+
+import Placeholder from "../resources/images/150.png";
+import Toppings from "./toppings";
+import Topping from "./topping";
```
Comment (steget) [inline client/src/components/editMenuItems.js:6]: not used, gets printed in console
```suggestion

```

File: client/src/components/editMenuItems.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                        <input
+                            className="ml-4 mt-3 mr-1 w-25 form-control-sm"
+                            type="text"
+                            placeholder="Pris utan tillägg..."
+                        ></input>
+                        <span>{"(kr)"}</span>
+
+                        <button
+                            type="button"
+                            class="btn btn-success btn-lg float-right m-3  pt-3 pb-3  "
```
Comment (steget) [inline client/src/components/editMenuItems.js:59]: className

File: client/src/components/editMenuItems.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                        >
+                            Spara
+                        </button>
+                        <button
+                            type="button"
+                            class="btn btn-danger btn-lg m-3 float-right  pt-3 pb-3  "
```
Comment (steget) [inline client/src/components/editMenuItems.js:65]: className

File: client/src/components/topping.js [hunk 5]
```diff
@@ -1,42 +1,42 @@
...
+                <div className="row align-items-center">
+                    <div className="col-8">{this.state.name}</div>
+                    <div className="col-4">{this.state.price} kr</div>
+                </div>
+            </div>
+        );
+    }
+}
+
+export default Topping;
```
Comment (steget) [inline client/src/components/topping.js:42]: I don't see any difference here at all?

Comment (Pear99) [reply client/src/components/topping.js:42]: Ska inte ha ändrat den men meckade med Topping ett tag.

**Pull request merged**